_pytest/main.py

```diff
@@ -100,11 +100,7 @@ def wrap_session(config, doit):
             session.exitstatus = EXIT_INTERNALERROR
             if excinfo.errisinstance(SystemExit):
                 sys.stderr.write("mainloop: caught Spurious SystemExit!\n")
-        else:
-            if session.testsfailed:
-                session.exitstatus = EXIT_TESTSFAILED
-            elif session.testscollected == 0:
-                session.exitstatus = EXIT_NOTESTSCOLLECTED
+
```

Comment:
is this change of moving this code somewhere else related to pytest cache integrastion?